src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useEffect, useState, ReactNode } from 'react';
+import { Device, AudioTransformDevice, VoiceFocusTransformDevice } from 'amazon-chime-sdk-js';
+
+import { ControlBarButton } from '../../ui/ControlBar/ControlBarButton';
+import { DeviceConfig } from '../../../types';
+import { isOptionActive, audioInputSelectionToDevice } from '../../../utils/device-utils';
+import { Microphone } from '../../ui/icons';
+import { PopOverItem } from '../../ui/PopOver/PopOverItem';
+import { PopOverSeparator } from '../../..';
+import { Spinner } from '../../ui/icons';
+import { useMeetingManager } from '../../../providers/MeetingProvider';
+import { useAudioInputs } from '../../../providers/DevicesProvider';
+import { useVoiceFocus } from '../../../providers/VoiceFocusProvider';
+import { useToggleLocalMute } from '../../../hooks/sdk/useToggleLocalMute';
+
+interface Props {
+  /** The label that will be shown when microphone is muted, it defaults to `Mute`. */
+  muteLabel?: string;
+  /** The label that will be shown when microphone is unmuted, it defaults to `Unmute`. */
+  unmuteLabel?: string;
+  /** Title attribute for the icon when muted, it defaults to `Muted microphone` in <Microphone />. */
+  mutedIconTitle?: string;
+  /** Title attribute for the icon when unmuted, it defaults to `Microphone` in <Microphone />. */
+  unmutedIconTitle?: string;
+}
+
+const AudioInputVFControl: React.FC<Props> = ({
+  muteLabel = 'Mute',
+  unmuteLabel = 'Unmute',
+  mutedIconTitle,
+  unmutedIconTitle
+}) => {
+  const meetingManager = useMeetingManager();
+  const [isVoiceFocusOn, setIsVoiceFocusOn] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isVoiceFocusEnabled, setIsVoiceFocusEnabled] = useState(false);
```

Comment:
`isVoiceFocusOn` and `isVoiceFocusEnabled` still is really close and builders or contributors may not differentiate it can we rename the `isVoiceFocusEnabled` to `isDeviceVoiceFocusEnabled` or `isDeviceVoiceFocusTransformDevice`?

Reply:
I prefer the current names, but thanks for your advice. I added some comments on both states, so customers can have a better understanding.